### ISSUE TITLE
Staging Review finding: Incorrect heading structure at multiple points in flow

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#96651]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** My Education Benefits
**Product:** 22-5490 Form
**Feature:** (Fry/DEA) Updates

## Findings details
**VA.gov Experience Standard - issue:** [User can't determine next steps in a flow.](https://design.va.gov/about/experience-standards/#user-can-determine-next-steps-in-a-flow)
**VA.gov Experience Standard - category:** [Findability](https://design.va.gov/about/experience-standards/#category-findability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility), @erinrwhite (IA)

### Description

There are multiple points in the flow where the heading structure doesn't match the actual structure of the content:
- On [/survivor-dependent-education-benefit-22-5490/introduction](https://staging.va.gov/education/survivor-dependent-education-benefit-22-5490/introduction) when both unauthenticated and authenticated, the H2 heading "Begin your application for education benefits" is followed immediately by another H2 heading, eg. "Save time---and save your work in progress..."
- On [/benefit-selection](https://staging.va.gov/education/survivor-dependent-education-benefit-22-5490/benefit-selection), there are two cards describing each of the two education benefits. The heading structure matches the visual style of the text but not the actual structure of the content.
- On [/review-personal-information](https://staging.va.gov/education/survivor-dependent-education-benefit-22-5490/review-personal-information), "Your personal information" is an H5 and the name is an H6, which doesn't fit with the hierarchy of the rest of the page and doesn't reflect the purpose of the name in this context.
- On [/contact-information](https://staging.va.gov/education/survivor-dependent-education-benefit-22-5490/contact-information), "We'll use this information to:" shouldn't be a heading.
- On [/direct-deposit](https://staging.va.gov/education/survivor-dependent-education-benefit-22-5490/direct-deposit), "Checking account" shouldn't be an H5 (and probably shouldn't be a heading)

Screen reader users often navigate by heading, and rely on the announced heading level to understand how different sections relate to one another. A heading structure that doesn't match the structure of the content can be confusing and disorienting.

## Recommended action

Work with an accessibility specialist to determine the correct heading structure at each of these points.

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** [WCAG 1.3.1 Info and Relationships - Level A](https://www.w3.org/TR/WCAG21/#info-and-relationships)
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
